packages/server/src/automations/steps/sendSmtpEmail.ts

```diff
@@ -48,6 +48,10 @@ export const definition: AutomationStepSchema = {
           type: AutomationIOType.STRING,
           title: "HTML Contents",
         },
+        attachments: {
+          type: AutomationIOType.STRING,
```

Comment:
Wondering if we should make this a new type, `URLS` - we need this to be a list of URLs to files that should be attached, might be useful from a validation point of view in future.

Also - how are the attachments named? Wondering if we actually need an `AutomationIOType.ATTACHMENT` - which is a specific type containing an array of objects `{ url: ..., name: ... }` as right now, I don't see how the attachment will be named against the email, this should probably be user configurable.